docs/tempo/website/troubleshooting/search-tag.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Tag search
+weight: 476
+---
+
+# Tag search
+
+An issue occurs while searching for traces in Grafana Explore. The **Service Name** and **Span Name** drop down lists are empty, and there is a `No options found` message.
+
+HTTP requests to Tempo query frontend endpoint at `/api/search/tag/service.name/values` would respond with an empty set.
+
+Note: this happens on Grafana Tempo 1.3 or higher.
```

Comment:
I had hoped that you would reword this, since it doesn't make sense to state that future versions of Tempo would have this issue.  They might not.

For now, let's just remove the reference.  I a reader is looking at the docs for a version of the product that has this issue, they'll see this section.
```suggestion
```